starter-code/Portfolio.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Joe Johnson|Portfolio</title>
+    <link rel="stylesheet" href="CSS/Styles.css">
+</head>
+
+<body>
+
+<header>
+<h1>Joe Johnson</h1>
+</header>
+
+<nav> 
+    <a href="About Me.html" >About Me</a>
+    <a href="Portfolio.html"> Portfolio</a>
+</nav>
+
+<div class="header">
```

Comment:
try to indent your code so that its easier to read

check this link for more information on indenting HTML https://www.granneman.com/webdev/coding/formatting-and-indenting-your-html